Minor fixes

Victor-GY-Yu opened the pull request.

Commit:
adding in a margin to help with the hidden axis label, and a wrap for longer service names (replace \n with <br> for plotly's more html based syntax)

Commit:
looks better with smaller font

Commit:
try and utilise extent of space available. Still looks a bit off for subpopulation/condition mixes with a lot of treatment pathways but it looks okay for most.

tomjemmett reviewed (approved):
awesome, I tried finding a fix to this and failed. Nice one! :-)

Victor-GY-Yu committed:
Merge branch 'master' into minor_fixes

tomjemmett committed:
fixes test module home

tomjemmett committed:
adds missing help page

tomjemmett committed:
updates test snaps

tomjemmett committed:
fixes test

tomjemmett committed:
adds missing imports

tomjemmett committed:
adds package

tomjemmett committed:
fixes lint issue

tomjemmett committed:
resolves conflicts

Merge branch 'minor_fixes' of https://github.com/The-Strategy-Unit/723_mh_covid_surge_modelling into minor_fixes

# Conflicts:
#	R/module_home.R
#	tests/testthat/_snaps/module_params.md

tomjemmett commented:
There are a few issues that have crept in when trying to rebase with master, so aborting this PR and creating a new one (#142 )
